goss.yaml

```diff
@@ -140,9 +140,10 @@ command:
   /usr/local/bin/docker-low-disk-gc:
     exit-status: 0
 
+  # test that we can build a docker image
   docker buildx build -f tests/Dockerfile --progress=plain -t buildkite-postgres:latest tests:
     exit-status: 0
-    timeout: 30000
+    timeout: 60000
```

Comment:
Interesting that I had to increase the timeout for m6gd.medium. I expect t4g w/o instance storage to be slower.